### ISSUE TITLE
openApiV2 and opaque JWT-like authentication tokens cause error

### ISSUE THREAD

**Issue by kevroletin** [wundergraph/wundergraph#818]
### Bug description

@wundergraph/sdk": "^0.143.1"

Hi, we are upgrading to OpenApiV2. I've successfully updated some of our api but I hit a problem with our KycStatusAuth api. Luckily that .graphql operation doesn't require changes between OpenApi and OpenApiV2, so it's easy to experiment.

There are two similar operations:
- KycStatus
- KycStatusAuth
The difference is that the later one get user email from claims and requires authentication.

```
|               | OpenApi | OpenApiV2 |
| KycStatus     | works   | works     |
| KycStatusAuth | works   | fails     |
```

Error message is not informative enough for me, so I can't identify the problem:

```
hooks pipeline failed: ResolveGraphQLResponse failed: Post "http://localhost:9992/openapis/kyc_status": json: error calling MarshalJSON for type json.RawMessage: invalid character '·' looking for beginning of value
```

---

 After investigation WG team reported that the problem is that our opaque bearer tokens look like a JWT token (because they have two dots '.'), but they arent.

### How to reproduce

I provided details in a private conversation

### Environment & setup

- OS: 
- Go version: 
- Database: 
- Node.js version: 

### WunderCtl Version

```

```

**Comment by Pagebakers** [comment]
Thanks for opening the issue. We will release a fix for this shortly. I'll update the issue once it's published.

**Comment by kevroletin** [comment]
It seems to be fixed in 0.145

**Comment by jensneuse** [comment]
> It seems to be fixed in 0.145

Thanks for confirming.
